doc/nrf/ug_nrf5340.rst

```diff
@@ -630,14 +630,29 @@ This container package can be used by update tools to pass both images during th
 Getting logging output
 **********************
 
-When connected to a computer, the nRF5340 DK emulates three virtual COM ports.
+When connected to a computer, the nRF5340 DK emulates virtual COM ports.
+The number of COM ports depends on the DK version you are using.
+
+nRF5340 DK v2.0.0 COM ports
+===========================
+
+When connected to a computer, the nRF5340 DK v2.0.0 emulates two virtual COM ports.
+In the default configuration, they are set up as follows:
+
+* The first COM port outputs the log from the network core (if available).
+* The second COM port outputs the log from the application core.
+
+nRF5340 DK v1.0.0 COM ports
+===========================
+
+When connected to a computer, the nRF5340 DK v1.0.0 emulates three virtual COM ports.
 In the default configuration, they are set up as follows:
 
 * The first COM port outputs the log from the network core (if available).
 * The second (middle) COM port is routed to the **P24** connector of the nRF5340 DK.
-* The third (last) COM port outputs the log from the application core sample.
+* The third (last) COM port outputs the log from the application core.
 
-You can use the second COM port as follows:
+To use the middle COM port in the nRF5340 DK v1.0.0, complete the following steps:
```

Comment:
NIT: Isn't it better to use second? Middle is in parentheses after all.

Reply:
I think "middle" is safer here, given that we use "second" in both paragraphs, and "middle" just in this second paragraph.